pkg/apply/applier.go

```diff
@@ -42,11 +44,6 @@ func NewApplier(factory util.Factory, ioStreams genericclioptions.IOStreams) *Ap
 	}
 }
 
-// poller defines the interface the applier needs to poll for status of resources.
-type poller interface {
-	Poll(ctx context.Context, identifiers []object.ObjMetadata, options polling.Options) <-chan pollevent.Event
-}
-
 // Applier performs the step of applying a set of resources into a cluster,
```

Comment:
We should probably put a couple paragraphs of detailed comments in the `applier` which describes how all the pieces work together in the "queue of tasks".

Reply:
I've added some more info here. I think the documentation in the taskrunner package does a decent job of explaining exactly how the tasks are executed.